src/app/pages/vm/vm-list/stop-vm-dialog/stop-vm-dialog.component.ts

```diff
@@ -34,20 +35,20 @@ export class StopVmDialogComponent {
       EntityJobComponent,
       {
         data: {
-          title: this.translate.instant('Stopping {rowName}', { rowName: this.vm.name }),
+          title: this.translate.instant('Stopping {rowName}', { rowName: this.data.vm.name }),
         },
       },
     );
-    jobDialogRef.componentInstance.setCall('vm.stop', [this.vm.id, {
+    jobDialogRef.componentInstance.setCall('vm.stop', [this.data.vm.id, {
       force: false,
       force_after_timeout: this.forceAfterTimeoutCheckbox.value,
     }]);
     jobDialogRef.componentInstance.submit();
-    jobDialogRef.componentInstance.success.pipe(untilDestroyed(this)).subscribe(() => {
+    jobDialogRef.componentInstance.success.pipe(untilDestroyed(this.data.parent)).subscribe(() => {
```

Comment:
Ideally, we should move this subscription into the parent `VmListComponent`, however I'm not sure if this is easily achievable